parliament.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop'
```

Comment:
We have added the versions based on the Gemfile.lock.  Not sure if this is what you intended?